src/frontend/apps/e2e/__tests__/app-impress/common.ts

```diff
@@ -4,6 +4,7 @@ export const CONFIG = {
   AI_FEATURE_ENABLED: true,
   CRISP_WEBSITE_ID: null,
   COLLABORATION_WS_URL: 'ws://localhost:4444/collaboration/ws/',
+  COLLABORATION_ONLY_CONNECTED_CAN_EDIT: false,
```

Comment:
`ONLY_CONNECTED` is confusing. I understood first only user being connected to the application. I would suggestion `COLLABORATION_ACTIVATED_WITHOUT_WS_CONNECTION` ? Is it more accurate ?